src/main/kotlin/com/mutualmobile/praxisspringboot/services/orgs/UserProjectService.kt

```diff
@@ -9,4 +10,8 @@ interface UserProjectService {
         projectId: String,
         userId: String
     ): ResponseEntity<ApiResponse<HarvestUserProject>>
+
+    fun findUserProject(projectId: String, userId: String): HarvestUserProject?
```

Comment:
@shubhamsinghmutualmobile The project is not linked to one userid, we should be able to search it by just the project id.